mkosi

```diff
@@ -2492,6 +2492,28 @@ def set_root_password(args: CommandLineArguments, root: str, do_run_build_script
             patch_file(os.path.join(root, 'etc/shadow'), jj)
 
 
+def run_prepare_script(args: CommandLineArguments, root: str, do_run_build_script: bool, cached: bool) -> None:
+    if args.prepare_script is None:
+        return
+    if cached:
+        return
+
+    verb = "build" if do_run_build_script else "final"
+
+    with complete_step('Running prepare script'):
+
+        # We copy the prepare script into the build tree. We'd prefer
+        # mounting it into the tree, but for that we'd need a good
+        # place to mount it to. But if we create that we might as well
+        # just copy the file anyway.
+
+        shutil.copy2(args.prepare_script,
+                     os.path.join(root, "root/prepare"))
```

Comment:
I think the general guideline for line length in mkosi is the same as systemd now (which is 109 columns) so this can be one line.